modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java

```diff
@@ -356,6 +362,164 @@ public void testInitializeFromFile() throws Exception {
+		AdminAccountGroupResource.Builder accountGroupResourceBuilder =
+			_adminAccountGroupResourcefactory.create();
+
+		AdminAccountGroupResource accountGroupResource =
+			accountGroupResourceBuilder.user(
+				_serviceContext.fetchUser()
+			).build();
+
+		AdminAccountGroup accountGroup1 =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP1");
+
+		Assert.assertNotNull(accountGroup1);
+		Assert.assertEquals("Test Account Group 1", accountGroup1.getName());
+
+		AdminAccountGroup accountGroup2 =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP2");
+
+		Assert.assertNotNull(accountGroup2);
+		Assert.assertEquals("Test Account Group 2", accountGroup2.getName());
+
+		AdminAccountGroup accountGroup3 =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP3");
+
+		Assert.assertNotNull(accountGroup3);
+		Assert.assertEquals("Test Account Group 3", accountGroup3.getName());
+	}
+
+	private void _assertAccountGroups2() throws Exception {
+		AdminAccountGroupResource.Builder accountGroupResourceBuilder =
+			_adminAccountGroupResourcefactory.create();
+
+		AdminAccountGroupResource accountGroupResource =
+			accountGroupResourceBuilder.user(
+				_serviceContext.fetchUser()
+			).build();
+
+		AdminAccountGroup accountGroup1 =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP1");
+
+		Assert.assertNotNull(accountGroup1);
+		Assert.assertEquals("Test Account Group 1", accountGroup1.getName());
+
+		AdminAccountGroup accountGroup2 =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP2");
+
+		Assert.assertNotNull(accountGroup2);
+		Assert.assertEquals(
+			"Test Account Group 2 Update", accountGroup2.getName());
+
+		AdminAccountGroup accountGroup3 =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP3");
+
+		Assert.assertNotNull(accountGroup3);
+		Assert.assertEquals("Test Account Group 3", accountGroup3.getName());
+
+		AdminAccountGroup accountGroup4 =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP4");
+
+		Assert.assertNotNull(accountGroup4);
+		Assert.assertEquals("Test Account Group 4", accountGroup4.getName());
+	}
+
+	private void _assertAccountGroupsAssignment1() throws Exception {
+		AccountEntry account =
+			_accountEntryLocalService.fetchAccountEntryByExternalReferenceCode(
+				"TESTACCOUNT1", _group.getCompanyId());
+
+		AdminAccountGroupResource.Builder accountGroupResourceBuilder =
+			_adminAccountGroupResourcefactory.create();
+
+		AdminAccountGroupResource accountGroupResource =
+			accountGroupResourceBuilder.user(
+				_serviceContext.fetchUser()
+			).build();
+
+		AdminAccountGroup accountGroup =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP1");
+
+		Assert.assertNotNull(account);
+
+		Assert.assertNotNull(accountGroup);
```

Comment:
to test the assignments you dont need to check the account and account group
you can apply the same logic as _assertRolesAssignments()